code/game/area/areas.dm

```diff
@@ -728,3 +732,17 @@ GLOBAL_LIST_EMPTY(teleportlocs)
 
 /area/get_virtual_z_level()
 	return get_virtual_z(get_turf(src))
+
+/// if it returns true, the mood effect assigned to the area is defined. Defaults to checking mood_job_allowed
+/area/proc/mood_check(mob/living/carbon/human/subject)
+	if(!mood_bonus)
+		return
+
+	. = TRUE
+
+	if(!length(mood_job_allowed))
+		return
```

Comment:
return .

Reply:
wait i thought it always returned period when it ran return unless you defined something to return

Reply:
just for code clarity